src/go/k8s/config/rbac/role.yaml

```diff
@@ -0,0 +1,575 @@
+
```

Comment:
This file `src/go/k8s/config/rbac/role.yaml` shold be removed from this commit ba8ddbbc43d3486f6f2b069191eace789f449e01

Reply:
The `role.yaml` should not be committed. The rbac changed its placement in https://github.com/redpanda-data/redpanda/commit/8bbfc6570e85b71db20102f341eade3dfe53ac7f
part of the https://github.com/redpanda-data/redpanda/pull/10791 commit.

Please see similar commit that removes that file

https://github.com/redpanda-data/redpanda/commit/1ca05f42c1c0b47193bdd2bb0df8a8b85fcbd0a3

Reply:
not sure how it got there but it's gone now.

Reply:
It's still here

Reply:
hehe... I checked each commit:

```
$ ls src/go/k8s/config/rbac/role.yaml 
ls: cannot access 'src/go/k8s/config/rbac/role.yaml': No such file or directory
```

...

and of course I didn't find it. I was in src/go/k8s. :facepalm: